source/_posts/2016-07-21-esp8266-and-micropython-part1.markdown

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title: "ESP8266 and MicroPython - Part 1"
+description: "Using MicroPython on ESP8266 based devices and Home Assistant."
+date: 2016-07-21 06:00:00 +0200
+date_formatted: "July 21, 2016"
+author: Fabian Affolter
+comments: true
+categories: How-To
+og_image: /images/blog/2016-07-micropython/social.png
+---
+
+<img src='/images/blog/2016-07-micropython/micropython.png' style='clear: right; border:none; box-shadow: none; float: right; margin-bottom: 12px;' width='200' />
+The first release of Micropython for ESP8266 was delivered a couple of weeks ago. The [documentation](http://docs.micropython.org/en/latest/esp8266/esp8266_contents.html) covers a lot of ground. This post is providing only a little summary which should get you started.
+
+If you don't participated in the KickStarter campaign only the source code was available. This meant that you needed to build the firmware on your own. As of now the [pre-built firmware](https://micropython.org/download/#esp8266) is available for the public.
```

Comment:
Until a couple of weeks ago, the pre-built MicroPython binary for the ESP8266 was only available to backers. This has changed now and it is available to the public for [download](https://micropython.org/download/#esp8266).